build.sh

```diff
@@ -154,14 +154,13 @@ function confirmRepo {
 if [ -z "$HTML_SOURCE" ]; then
   $QUIET || echo "HTML_SOURCE environment variable not set..."
   $QUIET || echo "OK, looking for HTML source file..."
-  PARENT_DIR=$(dirname $DIR)
-  if [ -f $PARENT_DIR/html/source ]; then
-    $QUIET || echo "OK, looked in the $PARENT_DIR/html directory and found HTML source there..."
-    HTML_SOURCE=$PARENT_DIR/html
+  if [ -f ../html/source ]; then
+    $QUIET || echo "OK, looked in the ../html directory and found HTML source there..."
```

Comment:
You're right. I don't want to mess with and thus have to test these code paths right now, so I've pushed a much simpler change.